docs/docs/config/bluetooth.md

```diff
@@ -9,7 +9,8 @@ See [Configuration Overview](index.md) for instructions on how to change these s
 
 ## Kconfig
 
-| Option                                | Type | Description                                                                                                                                  | Default |
-| ------------------------------------- | ---- | -------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `CONFIG_ZMK_BLE_PASSKEY_ENTRY`        | bool | Enable passkey entry during pairing for enhanced security. (Note: After enabling this, you will need to re-pair all previously paired hosts) | n       |
-| `CONFIG_BT_GATT_ENFORCE_SUBSCRIPTION` | bool | Low level setting for GATT subscriptions. Set to `n` to work around an annoying Windows bug with battery notifications.                      | y       |
+| Option                                | Type | Description                                                                                                                                                                                                                                            | Default |
+| ------------------------------------- | ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- |
+| `CONFIG_ZMK_BLE_BETA_TESTING`         | bool | Enables a combination of settings that are planned to be default in future versions of ZMK. This includes changes to timing on BLE pairing initation, BT Secure Connection passkey entry, and allowing overwrite of keys from previously paired hosts. | n       |
```

Comment:
Are we planning to do this one time, for these specific changes, or keep it in the future for other changes? The reason for the question is if it is the latter then we might want to not note the specifics here and perhaps link to the Kconfig file instead?

Reply:
I imagine us keeping it and tweaking/evolving it as we eventually migrate the ZMK defaults... Not sure it hurts to duplicate it here so it's just a tad more accessible, and we update both spots as needed.

Thoughts?

Reply:
If we are OK keeping this up to date as things change then that is likely the best solution.